en/flight_modes/return.md

```diff
@@ -15,26 +15,52 @@ The specific behaviour for each vehicle type is described below.
 
 ## Multi-Copter (MC)
 
-A copter/multi-rotor will first ascend to the `RTL_RETURN_ALT` altitude and then fly to the home position in a straight line (if already above `RTL_RETURN_ALT` it will return at its current altitude).
+A copter/multi-rotor will first ascend to the return altitude (*by default*, [RTL_RETURN_ALT](#RTL_RETURN_ALT)) and then fly to the home position in a straight line and constant altitude (if already above the return altitude it will return at its current altitude).

```

Comment:
HOpe this section makes sense. I wanted to start with the default behaviour rather than the new cone behaviour because it is less complex.

Reply:
@hamishwillee perfect.